topics/gradle-application-plugin.md

```diff
@@ -0,0 +1,67 @@
+[//]: # (title: Gradle Application plugin)
+
+<microformat>
+<p>
+<control>Sample project</control>: <a href="https://github.com/ktorio/ktor-gradle-sample/tree/main">ktor-gradle-sample</a>
+</p>
+</microformat>
+
+The Gradle [Application plugin](https://docs.gradle.org/current/userguide/application_plugin.html) provides the ability to package applications, including code dependencies and generated start scripts. In this topic, we'll show you how to package and run a Ktor application created in the [](Gradle.xml) topic.
+
+## Prerequisites {id="prerequisites"}
+Before starting this tutorial, do the following:
+* [Install IntelliJ IDEA](https://www.jetbrains.com/help/idea/installation-guide.html)
```

Comment:
Is it a necessary step?

Reply:
ok, I'll remove this requirement from tutorials